ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-release-1.9.yaml

```diff
@@ -54,6 +54,6 @@ tests:
       OO_TARGET_NAMESPACES: openshift-sandboxed-containers-operator
     workflow: optional-operators-ci-azure
 zz_generated_metadata:
-  branch: release-1.8
+  branch: release-1.9
```

Comment:
We don't have a release-1.9 branch yet... Should we create it first?
Or... this is all about building from the devel branch. Should it be "devel" here?

I'm actually wondering what this block mean :-/

Reply:
We have two files, one for devel and one for the release branch. I'm bumping everything here so that we don't forget. I'm assuming the one for the release branch will be not be used until we create the release branch.

Reply:
Oh ! Scratch that. You're editing the 1.8 build file, that makes our CI run on the release-1.8 branch.
So you're renaming/modifying it to run on an future release-1.9 branch.

As we're not pushing anything new on release-1.8, I'm fine moving this file away and reusing it for the next release.
Maybe prow will complain that the branch is not there though - let's see.

Reply:
My guess is that we should prepare prow first or it won't be able to run when we want to push our first PR to release-1.9...